app.py

```diff
@@ -55,36 +64,42 @@ async def decorated_function(*args, **kwargs):
 
 
 # Get All Users
-@app.get('/users')
+@app.get("/users")
 async def get_users():
     pool = await create_pool()
     async with pool.acquire() as conn:
         async with conn.cursor() as cur:
-            await cur.execute("SELECT * FROM users")
+            await cur.execute("SELECT user_id, user_name, email FROM users")
             users = await cur.fetchall()
             await cur.close()
             pool.close()
-            return jsonify({'users': users}), 200
+            users_dict_list = [dict(zip(("user_id", "user_name", "email", "password"), row)) for row in users]
```

Comment:
You can use `DictCursor`. It returns a dictionary as the response from a query.

https://aiomysql.readthedocs.io/en/latest/cursors.html#DictCursor